src/Components/ADT3DSceneBuilder/Internal/VisualRuleForm/Internal/ConditionsList.tsx

```diff
@@ -106,30 +158,40 @@ const ConditionsList: React.FC<IConditionsListProps> = (props) => {
     // Update list everytime valueRanges and expressionType change
     useEffect(() => {
         setConditions(getConditionItems(valueRanges, expressionType));
-    }, [valueRanges, expressionType]);
-
-    // Callbacks
-    const handleOpenFlyout = useCallback(() => {
-        // TODO: Open callout
-        alert('New condition');
-    }, []);
+    }, [valueRanges, expressionType, getConditionItems]);
 
     return (
-        <div className={classNames.container}>
-            <Stack>
-                <CardboardList<Condition>
-                    listKey={LIST_KEY}
-                    items={conditions}
+        <>
+            <div className={classNames.root}>
+                <Stack>
+                    <CardboardList<Condition>
```

Comment:
What happens when I add overlapping ranges for conditions? Also the visual representation of between using "-" might be confusing since we allow negative numbers: For example I added between -100 and 10 and -10 and 100 as two conditions but it seems a little confusing in the list since there is another dash is used to represent min and max values between. What do you think about this @azimamansuri ?
![image](https://user-images.githubusercontent.com/45217314/198155872-31fedc09-2544-4d4f-a5d3-3d5b8712fadb.png)

Reply:
Overlapping ranges are allowed, it was part of the update. The dash is something I'll take up with Azima, good catch.